plugins/pbjsGlobals.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(api, options) {
         const modName = getModuleName(state.filename);
         if (modName != null) {
           // append "registration" of module file to $$PREBID_GLOBAL$$.installedModules
-          path.node.body.push(...api.parse(`window.${pbGlobal}.installedModules.push('${modName}');`).program.body);
+          path.node.body.push(...api.parse(`window.${pbGlobal}.installedModules.push('${modName}');`, {filename: modName}).program.body);
```

Comment:
Thanks for figuring this out! 
The actual filename is under `state.filename`. Reading through https://babeljs.io/docs/en/options#filename it seems better to use that, because I can imagine a situation where `modName` clashes with some actual file in your build and babel makes the wrong decision on it.

Reply:
That makes sense! Haven't tested it yet, but what do you think about appending the `modName` to the `state.filename` so each module keeps its own name? `'${state.filename}.${modName}'`. I don't know if having all modules with the same file name creates some conflict in this case.

Reply:
this piece of code runs once for each module file, so that should not be a problem - `state.filename` will be different for each instance.

Reply:
ab779edc95032fb36940aeb2eac03ad1dc95bc11 updates the filename and the build continues to work as expected locally